lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -159,6 +159,23 @@ class SessionStateEnforcer {
       std::unique_ptr<SessionState>& session, SessionFsmState target_state,
       SessionStateUpdateCriteria* session_uc);
 
+  /*
+   * Acquire and update session rules based on the IMSI
+   *
+   * */
+  uint32_t update_session_rules(
+      std::unique_ptr<SessionState>& session, bool gnb_teid_get,

```

Comment:
Commented somewhere else too, but let's rename the boolean flags to `get_*`

Reply:
updated the parameters as suggested
gnb_teid_get -> get_gnb_teid
upf_teid_get -> get_upf_teid